azure-toolkit-libs/azure-toolkit-resource-lib/src/main/java/com/microsoft/azure/toolkit/lib/resource/AzureGroup.java

```diff
@@ -35,44 +37,63 @@ private AzureGroup(@Nonnull final List<Subscription> subscriptions) {
         super(AzureGroup::new, subscriptions);
     }
 
-    public List<ResourceGroupEntity> list() {
+    public List<ResourceGroup> list() {
         return Flux.fromIterable(getSubscriptions()).parallel()
                 .map(subscription -> getResourceManager(subscription.getId()))
                 .flatMap(azureResourceManager -> azureResourceManager.resourceGroups().listAsync())
-                .map(ResourceGroupEntity::fromResource)
+                .map(AzureGroup::fromResource)
                 .sequential().collectList().block();
     }
 
-    public ResourceGroupEntity getById(@Nonnull String idStr) {
+    public List<ResourceGroup> list(String subscriptionId) {
+        return getResourceManager(subscriptionId).resourceGroups().listAsync()
+                .map(AzureGroup::fromResource)
+                .collect(Collectors.toList()).block();
+    }
+
+    public ResourceGroup getById(@Nonnull String idStr) {
         final ResourceId id = ResourceId.fromString(idStr);
         return get(id.subscriptionId(), id.resourceGroupName());
     }
 
-    public ResourceGroupEntity getByName(@Nonnull String name) {
+    public ResourceGroup getByName(@Nonnull String name) {
         return get(getDefaultSubscription().getId(), name);
     }
 
-    public ResourceGroupEntity get(@Nonnull String subscriptionId, @Nonnull String name) {
-        return ResourceGroupEntity.fromResource(getResourceManager(subscriptionId).resourceGroups().getByName(name));
+    public ResourceGroup get(@Nonnull String subscriptionId, @Nonnull String name) {
```

Comment:
get what?
It will be better to naming it more detail.

Reply:
equals: az account show(we change the term 'show' to 'get')